### PR TITLE
#115 Feature/enable linting

### LINKED ISSUE #115: Uncomment `golangci-lint` pre-commit hook
Uncomment the hook after fixing the golang linting errors

`golangci-lint` results:

image

_Originally posted by @RothAndrew in https://github.com/defenseunicorns/zarf/issues/112#issuecomment-946268723_

### PR DESCRIPTION
**Pull request by matt-strong**
Still need to fix regexp lint issue

Fixes #115

### CODE REVIEW

File: cli/cmd/root.go [hunk 1]
```diff
@@ -64,6 +64,6 @@ func setLogLevel(logLevel string) {
 	case "panic":
 		logrus.SetLevel(logrus.PanicLevel)
 	default:
-		logrus.Fatal("Unrecognized log level entry: %s", logLevel)
```
Comment (jeff-mccoy) [inline cli/cmd/root.go:67]: This looks like a typo, should be FatalF() I believe.

File: cli/internal/packager/inspect.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ func Inspect(packageName string) {
 	utils.ColorPrintYAML(text)
 
 	// Load the config to get the build version
-	config.LoadConfig(tempPath.base + "/zarf.yaml")
+	_ = config.LoadConfig(tempPath.base + "/zarf.yaml")
```
Comment (jeff-mccoy) [inline cli/internal/packager/inspect.go:35]: We should handle an error here or they could get a crash

**Review by jeff-mccoy — CHANGES_REQUESTED** [review]
Looks good after addressing the comments, thanks

File: cli/cmd/root.go (hunk 1 above)
Comment (YrrepNoj) [reply cli/cmd/root.go:67]: logrus.Fatalf( .... )

Comment (matt-strong) [reply cli/cmd/root.go:67]: thanka

File: cli/internal/packager/inspect.go (hunk 2 above)
Comment (matt-strong) [reply cli/internal/packager/inspect.go]: copied what inspect.go does on an error loading config

File: cli/cmd/root.go [hunk 3]
```diff
@@ -64,6 +64,6 @@ func setLogLevel(logLevel string) {
...
+		logrus.Fatalf("Unrecognized log level entry:", logLevel)
```
Comment (jeff-mccoy) [inline cli/cmd/root.go:67]: `logrus.Fatalf("Unrecognized log level entry: %s", logLevel)` and it should be g2g, would have just fixed online but think that would bork our gpg signature requirement.

**Review by jeff-mccoy — CHANGES_REQUESTED** [review]

**Review by jeff-mccoy — APPROVED** [review]

**Pull request merged**